CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Updated main and distro READMEs
     ([#205](https://github.com/microsoft/ApplicationInsights-Python/pull/205))
+- Update CONTRIBUTING.md, support Py3.11
+    ([#210](https://github.com/microsoft/ApplicationInsights-Python/pull/210))
```

Comment:
Are we adding to the bottom instead of the top?

Reply:
I am indifferent.